feature/src/main/java/org/woowatechcamp/banchan/feature/custom/CartBottomSheetFragment.kt

```diff
@@ -50,6 +52,11 @@ class CartBottomSheetFragment : BottomSheetDialogFragment() {
             btnCartBottomSheetCancel.setOnClickListener { this@CartBottomSheetFragment.dismiss() }
             btnCartBottomSheetMinus.setOnClickListener { if (countFlow.value > 1) countFlow.value-- }
             btnCartBottomSheetPlus.setOnClickListener { if (countFlow.value < 99) countFlow.value++ }
+            btnCartBottomSheetAdd.setOnClickListener {
+                // TODO: add item to cart logic
+                DialogUtil.showCartAdditionConfirmPopup<PurchaseActivity>(requireActivity())
```

Comment:
> 확장함수 형식으로 표현해도 간결할 것 같네요!

이거 참고해서

```suggestion
                requireActivity().showCartAdditionConfirmPopup<PurchaseActivity>()
```
이렇게 해도 괜찮을 것 같아요!

Reply:
같이 반영하겠습니다.